src/book.cpp

```diff
@@ -220,4 +220,16 @@ bool Book::getTagBool(const std::string& tagName) const {
   return convertStrToBool(getTagStr(tagName));
 }
 
+std::string Book::getCategory() const
+{
+  try
+  {
+    return getTagStr("category");
+  }
+  catch ( const std::out_of_range& )
+  {
+    return "unknown";
```

Comment:
Do we really want to have "unknown" ?
Maybe we can use "other", or a empty string, or skip the node ?